x-pack/plugins/alerting/server/usage/alerts_telemetry.test.ts

```diff
@@ -106,11 +106,21 @@ Object {
   "count_rules_namespaces": 0,
   "count_total": 4,
   "schedule_time": Object {
+    "avg": "4.5s",
+    "max": "10s",
+    "min": "1s",
+  },
+  "schedule_time_number": Object {
```

Comment:
Are these always in seconds? Maybe we can name it `schedule_time_s` so the unit is clear from the field name?

Reply:
Yeah, it always in seconds, as a corresponding schedule_time, but the number representation. But probably you are right, maybe something like schedule_time_number_s?